tests/e2e/gc_rwx_reclaim_policy.go

```diff
@@ -0,0 +1,518 @@
+		framework.RunKubectlOrDie(namespace, wrtiecmd2...)
+		output = framework.RunKubectlOrDie(namespace, cmd2...)
+		gomega.Expect(strings.Contains(output, "Hello message from test into Pod2")).NotTo(gomega.BeFalse())
+	})
+
+	/*
+		Verify static volume provisioning fails
+		1. Create a Storage class
+		2. Create a PVC-1 with "ReadWriteOnce" using the SC from in the GC
+		3. Wait for PVC-1 to be Bound in the GC
+		4. Verify if the mapping PVC is bound in the SV cluster using the volume handler
+		5. Verify CnsVolumeMetadata CRD is created
+		6. Verify health status of PVC
+		7. Verify volume is created on CNS by using CNSQuery API and also check metadata is pushed to CNS
+		8. Create a PV-2 with "ReadWriteMany" access mode in GC using the bound PVC-1 as the volumeId in the SV cluster
+		9. Create PVC2 with "ReadWriteMany" access mode which gets bounds to PV-2 (reclaim policy=delete)
+		10. Verify CnsVolumeMetadata CRD is created
+		11. Verify health status of PVC
+		12. Verify volume is created on CNS by using CNSQuery API and also check metadata is pushed to CNS
+		13. Create Pod using PVC-2 created above
+		14. Verify Pod creation fails
+		15. Verify the error message returned on failure is correct
+		16. Delete PVC and PVs from GC
+		17. Verify if PVC and PVs are deleted in the SV cluster and GC
+		18. Verify CnsVolumeMetadata CRD is deleted
+		19. Check if the VolumeID is deleted from CNS by using CNSQuery API
+	*/
+	ginkgo.It("Verify ReadWriteOnce PVC is not usable as ReadWriteMany", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		var storageclasspvc *storagev1.StorageClass
+		var pvclaim *v1.PersistentVolumeClaim
+		var err error
+
+		ginkgo.By("CNS_TEST: Running for GC setup")
+		scParameters[svStorageClassName] = storagePolicyName
+		ginkgo.By("Creating a PVC")
+		storageclasspvc, pvclaim, err = createPVCAndStorageClass(client,
+			namespace, nil, scParameters, diskSize, nil, "", false, v1.ReadWriteOnce)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		defer func() {
+			err = client.StorageV1().StorageClasses().Delete(ctx, storageclasspvc.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		ginkgo.By("Expect claim to provision volume successfully")
+		persistentvolumes, err := fpv.WaitForPVClaimBoundPhase(client,
+			[]*v1.PersistentVolumeClaim{pvclaim}, framework.ClaimProvisionTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Failed to provision volume")
+
+		volHandle := persistentvolumes[0].Spec.CSI.VolumeHandle
+		gomega.Expect(volHandle).NotTo(gomega.BeEmpty())
+		volumeID := getVolumeIDFromSupervisorCluster(volHandle)
+		gomega.Expect(volumeID).NotTo(gomega.BeEmpty())
+
+		defer func() {
+			err = fpv.DeletePersistentVolumeClaim(client, pvclaim.Name, pvclaim.Namespace)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			err = e2eVSphere.waitForCNSVolumeToBeDeleted(volHandle)
```

Comment:
volumeID

Reply:
fixed

Reply:
Not seeing the change in the diff, please recheck